src/postgres/tree/PostgresDatabaseTreeItem.ts

```diff
@@ -54,7 +55,7 @@ export class PostgresDatabaseTreeItem extends AzureParentTreeItem<ISubscriptionC
 
     public async loadMoreChildrenImpl(_clearCache: boolean, context: IActionContext): Promise<AzExtTreeItem[]> {
         try {
-            const clientConfig: ClientConfig = await getClientConfig(this.parent, this.databaseName);
+            const clientConfig: ClientConfig = await checkAuthentication(context, this);
```

Comment:
This should still be `getClientConfig` so that the errors are shown in the tree (the whole point of the `catch` block), as opposed to warning notifications like in `checkAuthentication`